Tests/UpstreamIntegrationTests/ConfigOverrideTests.swift

```diff
@@ -45,7 +45,7 @@ class ConfigOverrideTests: TestBase, AnyCodableAsserts {
             print("Extensions registration is complete")
             waitForRegistration.countDown()
         })
-        XCTAssertEqual(DispatchTimeoutResult.success, waitForRegistration.await(timeout: 2))
+        XCTAssertEqual(DispatchTimeoutResult.success, waitForRegistration.await(timeout: 30))
```

Comment:
we need timeout '30' here?  Can it be lower?

Reply:
That makes sense.

Reply:
@timkimadobe this will wait for entire 30 seconds right? We are increasing the timeout 15X. Can we keep 10 seconds and see if that helps with our processing. Are you aware of the time it is actually taking currently?